src/main/fc/core.c

```diff
@@ -1020,7 +1036,7 @@ void processRxModes(timeUs_t currentTimeUs)
     }
 
 #ifdef USE_ALT_HOLD_MODE
-    // only if armed
+    // only if armed; can coexist with position hold
     if (ARMING_FLAG(ARMED) 
         // and either the alt_hold switch is activated, or are in failsafe
         && (IS_RC_MODE_ACTIVE(BOXALTHOLD) || failsafeIsActive())
```

Comment:
Yes it's not very obvious...
The next line blocks Altitude Hold if we are in GPS Rescue Mode (whether by Mode switch, or failsafe).
Considering Failsafe, we have three options - drop, landing mode or gps rescue.
If we are in Failsafe and Drop mode, quad is now disarmed, so can't get here.
If we are in Failsafe and GPS Rescue mode, the next line will block Altitude Hold
Hence the only mode left is Landing Mode.

I'll move the GPS Rescue lock up one line, I think that may make it a bit more clear.

Also might as well enable Position Hold in Failsafe Landing Mode also, because that will stop drift in FS landing mode - making it quite usable!  Hadn't thought of that until your comment.  Thanks again!

Reply:
> Also might as well enable Position Hold in Failsafe Landing Mode also

it could be separate option - not land but hold position at home

Reply:
Yes, if we add another failsafe mode, aka 'hold position', then we will need to change this, but for now it is economical.